corehq/apps/app_manager/models.py

```diff
@@ -3507,6 +3498,31 @@ def get_filter_value(self, user, ui_filter):
         return None
 
 
+class AncestorLocationTypeFilter(ReportAppFilter):
+    ancestor_location_type_name = StringProperty()
+
+    def get_filter_value(self, user, ui_filter):
```

Comment:
should this just return a single value? I *think* that you can only have a single ancestor of any given type for a given location. @esoergel could confirm.